features/admin.server-configurations.v1/forms/login-attempt-security-form.tsx

```diff
@@ -425,7 +445,7 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
     }
 
     return (
-        <div className={ "connector-form" }>
+        <div className={ "connector-form login-attempt-security-form" }>
```

Comment:
Redundant `{` brace.

```suggestion
        <div className="connector-form login-attempt-security-form">
```

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/6997/commits/6370a688cb4b5e6c89e890caca37aa6066bdfad3